android/app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
   <uses-permission android:name="android.permission.INTERNET" />
   <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
   <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+  <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
for context, think this is automatically added via expo-location but explicitly added here for clarity

https://github.com/expo/expo/tree/master/packages/expo-location#configure-for-android